dev.mateusz1913.Pilot.json

```diff
@@ -0,0 +1,58 @@
+{
+    "id" : "dev.mateusz1913.Pilot",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "pilot",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=host"
```

Comment:
why does this need `host` filesystem access?

Reply:
The app uses [Android Debug Bridge](https://developer.android.com/tools/adb) to communicate with Android TV emulators and simulators. It's installed as a part of Android development SDK somewhere in host's filesystem.

Initially instead of requesting access to host filesystem I tried using open file/directory portal to get access to whatever path user used to install Android SDK but then I could not use it to run `adb` executable. I also tried to use `flatpak-spawn` which worked, however it was flagged by flatpak-builder-linter, so I removed it and replace it with `--filesystem=host` (this one passed linter)

Reply:
>  It's installed as a part of Android development SDK somewhere in host's filesystem.

No. NEVER RUN a binary in the host system. This is not how sandboxing work. If you need these tools they should be added to the manifest and part of the flaptak.

Reply:
As for flatpak-spawn you need to request an exception but requiring to install the "developer tools" to use the flatpak is kinda counter point of why flatpak. Include adb in the flatpak.

Reply:
Unfortunately it's impossible to include Android SDK due to the licensing it has. When trying to download [the standalone release](https://developer.android.com/tools/releases/platform-tools) it displays the terms and conditions that explicitly mention in points 2.1 and 2.2 that user needs to accept those, which would be not possible if I'd include it in the flatpak:
```
2. Accepting this License Agreement
2.1 In order to use the SDK, you must first agree to the License Agreement. You may not use the SDK if you do not accept the License Agreement. 2.2 By clicking to accept and/or using this SDK, you hereby agree to the terms of the License Agreement. 2.3 You may not use the SDK and may not accept the License Agreement if you are a person barred from receiving the SDK under the laws of the United States or other countries, including the country in which you are resident or from which you use the SDK. 2.4 If you are agreeing to be bound by the License Agreement on behalf of your employer or other entity, you represent and warrant that you have full legal authority to bind your employer or such entity to the License Agreement. If you do not have the requisite authority, you may not accept the License Agreement or use the SDK on behalf of your employer or other entity. 
```
Also in point 3.4 they mention that copying and redistributing of the SDK is prohibited
```
3. SDK License from Google
3.1 Subject to the terms of the License Agreement, Google grants you a limited, worldwide, royalty-free, non-assignable, non-exclusive, and non-sublicensable license to use the SDK solely to develop applications for compatible implementations of Android. 3.2 You may not use this SDK to develop applications for other platforms (including non-compatible implementations of Android) or to develop another SDK. You are of course free to develop applications for other platforms, including non-compatible implementations of Android, provided that this SDK is not used for that purpose. 3.3 You agree that Google or third parties own all legal right, title and interest in and to the SDK, including any Intellectual Property Rights that subsist in the SDK. "Intellectual Property Rights" means any and all rights under patent law, copyright law, trade secret law, trademark law, and any and all other proprietary rights. Google reserves all rights not expressly granted to you. 3.4 You may not use the SDK for any purpose not expressly permitted by the License Agreement. Except to the extent required by applicable third party licenses, you may not copy (except for backup purposes), modify, adapt, redistribute, decompile, reverse engineer, disassemble, or create derivative works of the SDK or any part of the SDK. 3.5 Use, reproduction and distribution of components of the SDK licensed under an open source software license are governed solely by the terms of that open source software license and not the License Agreement. 3.6 You agree that the form and nature of the SDK that Google provides may change without prior notice to you and that future versions of the SDK may be incompatible with applications developed on previous versions of the SDK. You agree that Google may stop (permanently or temporarily) providing the SDK (or any features within the SDK) to you or to users generally at Google's sole discretion, without prior notice to you. 3.7 Nothing in the License Agreement gives you a right to use any of Google's trade names, trademarks, service marks, logos, domain names, or other distinctive brand features. 3.8 You agree that you will not remove, obscure, or alter any proprietary rights notices (including copyright and trademark notices) that may be affixed to or contained within the SDK. 
```

In that case, it seems the app will never be able to satisfy the sandboxing. I'll decline the PR, thanks for your time and reviews